todocore/src/main/java/com/knighttodo/todocore/exception/FindDayByDateException.java

```diff
@@ -0,0 +1,8 @@
+package com.knighttodo.todocore.exception;
+
+public class FindDayByDateException extends RuntimeException {
```

Comment:
no @ResponseCode annotation, add it please

Reply:
Done!

Reply:
Consider renaming it to DayByDateNotFoundException

Reply:
Ok.